Tests/Unit/MSFT_xIISFeatureDelegation.tests.ps1

```diff
@@ -4,14 +4,30 @@ $global:DSCResourceName = 'MSFT_xIISFeatureDelegation'
 #region HEADER
 
 [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
- if ( (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests'))) -or `
-      (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests\TestHelper.psm1'))) )
-{
-    & git @('clone','https://github.com/PowerShell/DscResource.Tests.git',(Join-Path -Path $moduleRoot -ChildPath '\DSCResource.Tests\'))
+$repoSource = (Get-Module -Name $global:DSCModuleName -ListAvailable)
+
+if ($repoSource.RepositorySourceLocation.Host -eq 'www.powershellgallery.com') {
+    if ( -not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DscResourceTestHelper')) ) {
+        #install test folders from gallery
+        Save-Module -Name 'DscResourceTestHelper' -Path $moduleRoot
+    }
+
+    $testModuleVer = '0.2.0.0'
+    Import-Module (Join-Path -Path $moduleRoot -ChildPath "DscResourceTestHelper\$testModuleVer\TestHelper.psm1") -Force
+} 
+elseif ( (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DscResource.Tests'))) -or `
+        (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DscResource.Tests\DscResourceTestHelper'))) -or `
+       (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DscResource.Tests\DscResourceTestHelper\TestHelper.psm1'))) ) {
```

Comment:
you want to confirm the path DscResource.Tests\DscResourceTestHelper\TestHelper.psm1 exists... if you are checking for the leave, if any of the parents doesn't exist it will fail as well, so this will do the job